src/components/Form/ValidatorMap.js

```diff
@@ -0,0 +1,16 @@
+import validateMinMaxFactory from '../../validators/validateMinMax'
+import validateTruthy from '../../validators/validateTruthy'
+
+export default function GenerateValidator(validatorName, args) {
+  switch (validatorName) {
+    case 'truthy':
+      return validateTruthy
+    case 'minMax':
+      return validateMinMaxFactory.apply(null, args)
+    case 'exampleEvenNumber':
```

Comment:
Throwaway code for the purposes of illustrating what this setup can do.

Reply:
Yeah, I'd say my comments for the "mappers" and "validators" can be considered the same. I don't want us to have to manage/maintain all these. I think it's enough for us to provide some boiler-plate validators, but this generation/builder type of stuff belongs higher up layer and on the client side to my mind.

Reply:
sounds good